blind_75/reorder_list.py

```diff
@@ -0,0 +1,74 @@
+# https://leetcode.com/problems/reorder-list/ 
+# problem 143 Linked List 
+# You are given the head of a singly linked-list. The list can be represented as:
+
```

Comment:
@marktai 

This one will be hard to follow but here's a LC hard problem which is really just 3 LC easy/mediums in one. I think had I been more clear and thorough in problem solving I would have had a chance